src/Components/Shared/PlayerButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { Button } from "react-bootstrap"
+import { TrackSymbols } from "../../Constants/Symbols"
+import * as PlayerEndpoints from "../../Endpoints/Player"
+
+interface PlayerButtonProps {
+    currentlyPlaying: Boolean
+}
+
+interface PlayerButtonState extends PlayerButtonProps {}
+
+export default class PlayerButton extends React.Component<PlayerButtonProps, PlayerButtonState> {
+    constructor(props: PlayerButtonProps) {
+        super(props)
+
+        this.state = {
+            ...props,
+        }
+    }
+
+    playerButtonClicked() {
+        this.state.currentlyPlaying ? PlayerEndpoints.pause : PlayerEndpoints.startResume
```

Comment:
Do you need `()` after the function names here? Is this actually calling these functions or just referencing them?

Reply:
Shit you right. I'll add those